src/openrct2/core/String.cpp

```diff
@@ -745,6 +745,50 @@ namespace String
 #endif
     }
 
+    std::string ToLower(const std::string_view& src)
+    {
+#ifdef _WIN32
```

Comment:
```suggestion
#if defined(_WIN32) && _WIN32_WINNT >= 0x060
```

Reply:
That would break compilation for WINNT5.1.

It should be noted that this function is a straight copy of the `ToUpper` function, with the relevant bits changed.

Reply:
So for NT 5.1 systems you never get a lowercase?

Reply:
Indeed. Unless someone were to implement it.

Reply:
I think identifiers should be case sensitive or invariant culture ToLower. This means you want the implementation to be same regardless of the user's culture. A simple ASCII implementation may be best.